librz/core/cautocmpl.c

```diff
@@ -444,6 +490,18 @@ static void autocmplt_cmd_arg(RzCore *core, RzLineNSCompletionResult *res, const
 	case RZ_CMD_ARG_TYPE_FLAG:
 		autocmplt_cmd_arg_flag(core, res, s, len);
 		break;
+	case RZ_CMD_ARG_TYPE_ENUM_TYPE:
```

Comment:
no `alias` and `class` autocompletion?

Reply:
Classes aren't implemented very well yet, it will be added after RzTypes refactoring finished. Aliases support will add right now.

Reply:
Added typedefs support.